package.json

```diff
@@ -36,6 +36,9 @@
     "uuid": "^8.3.2",
     "winston": "^3.1.0"
   },
+  "optionalDependencies": {
```

Comment:
what will happen if user installs using `npm install --omit=optional`? Do we handle such a case?

Reply:
I added try catch just in case when users do not install the 'keytar' library. In this case, the driver returns null, and they can see the warning message in the driver logs.

Reply:
As we decided not to use 'keytar' library, this conversation was resolved.